resources/public/api/conf/3.0/retrieve_property_bsas.raml

```diff
@@ -1,11 +1,13 @@
 is:
   - headers.acceptJson
   - headers.testScenarioHeader
+  - queryParameters.taxYearTys
   - errors.formatNino
   - errors.formatCalculationId
   - errors.ruleTypeOfBusinessIncorrectUk
   - errors.clientOrAgentNotAuthorised
   - errors.summaryNotFound
+  - errors.ruleTaxYearNotSupported
```

Comment:
Think you might also need these
  - errors.formatTaxYear
  - errors.ruleTaxYearRangeInvalid
  - errors.invalidTaxYear